pythnet/pythnet_sdk/src/test_utils/mod.rs

```diff
@@ -67,6 +78,18 @@ pub const DEFAULT_VALID_TIME_PERIOD: u64 = 180;
 
 const DEFAULT_SEQUENCE: u64 = 2;
 
+const NUM_GUARDIANS: u8 = 19;
+const NUM_SIGNATURES: usize = 13;
+
+pub fn create_dummy_guardians() -> Vec<SecretKey> {
+    let mut result: Vec<SecretKey> = vec![];
+    for i in 0..NUM_GUARDIANS {
+        let mut secret_key_bytes = [0u8; 32];
+        secret_key_bytes[0] = i + 1;
+        result.push(SecretKey::parse(&secret_key_bytes).unwrap());
+    }
+    result
+}
```

Comment:
the name of this method sounds like it could return a different guardian set each time. Suggest calling it `dummy_guardians` instead